src/bin/generate-sample-proof.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024, Horizen Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
Run this script generate `proof.bin`, `pubs.bin` and `vk.bin` in the root: can you add them to `.gitignore`?